client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -735,6 +752,30 @@ class ContentDataProvider
         }
       : undefined;
   }
+
+  private async getCachedParentPath(item: ContentItem): Promise<string> {
```

Comment:
Since we're really using this to get tooltip information, it might be worth updating this to be `getItemTooltip` and calling `tooltip: this.getItemTooltip(item)` up above